### PR TITLE
[CSS] Hide `new-user-avatar-cta` protip box

### PR DESCRIPTION
**Pull request by zackschuster**
Hides the annoying 'Pro Tip! Updating your profile...' box on the personal profile page (appears whenever your name isn't set)

### CODE REVIEW

File: extension/content.css [hunk 1]
```diff
@@ -514,3 +514,7 @@ hidden content area created to increase hover target
 .btn-mark-unread {
 	margin-top: 8px;
 }
+
+.new-user-avatar-cta {
```
Comment (sindresorhus) [inline extension/content.css]: Should have a comment description like other overrides.

**Comment by zackschuster** [comment]
Fixed and squashed. Unfortunately, I'm at work right now and working around our bizarre proxy settings is proving too much, so I can't update to HEAD. I can do that when I get home.

**Comment by zackschuster** [comment]
okay, i've updated to master. should be good to go now.

**Comment by sindresorhus** [comment]
Look at the diff. You resolved the conflict incorrectly. It's now invalid CSS.

**Comment by zackschuster** [comment]
whoops. did a pull from your master and didn't get a merge conflict, so i assumed it was fine. my apologies!

**Pull request merged**